xeno/shields.md

```diff
@@ -0,0 +1,112 @@
+
+Buff abilities are explained further below. Shields are kept for castes that only give them to themselves.
+
+Shields are integrated, in some cases, into these buff abilities.
+
+## Goals
+
+1. To replace shields with buff abilities
+
+2. To make said buff abilities intuitive and well telegraphed
+
+3. To make said buff abilities increase survivability and durability but not persist off the front line / long after they are applied.
+
+4. To use a variety of buff abilities that are not underpowered or essential for engagements (xenos should not rely on them)
+
+5. To not add buff abilities to new castes that did not have them previously.
+
+## Non-goals
+
+1. Majorly affect side balance. This design does not aim to make a side stronger or weaker with this, which is why every caste gets compensatory buffs.
+
+2. Make warrior too squishy on the front-line and unable to stay there for any longer amount of time due to a lack of buff ability or shield
+
+3. Majorly handhold xenos with these buffs or make them required for engagements (think xenos afking in the hive for maturity - we don't want xenos to have to afk for buffs before going into a fight)
+
+4. Repeat the same buff abilities over and over on different xenos. When a xeno needs a shield-like buff ability, it should be unique to some extent. 
+
+5. Make said buff abilities overpowered and frustrating to fight.
+
+## Content
+
+Shields to be replaced in most areas by buff abilities.
+
+Buff abilities to be temporary, somewhat increase durability or survivability, or give a resistance to a xeno. They are not to be heals. 
+
+Varied types of buff ability to be made (clarity, speed boosts, etc) and given to different castes where they best fit.
+
+Buff abilities to be limited to praetorians, carriers (shaman), the queen, drone (gardener), and potentially other castes that already have buff abilities. Shaman carrier and Warden prae **must** recieve new buff abilities or offensive abilities to compensate for their loss of shields.
+
+Castes that have buff abilities that only buff themselves are not to recieve buff abilities that can buff other xenos.
+
+Shields, or buffs like them, exist to give xenos more durability and strength - basically to increase their survivability not in the form of a simple heal. It should be clear to players when this xeno is more durable, and should also not persist off the frontline or majorly interfere with how you fight a specific caste when they are applied. 
+
+A "buff ability" is a replacement for shields, and is the name that is used to refer to any ability that can replace shields.
+
+Clarity stacks are limited effect stacks that let xenos resist minor stuns, can be seen on boxer warrior. These can be used as replacement buffs for shields
+
+Warden is a xeno that can provide, as one of its abilities, short-term buffs or heals. (not long term spam shields)
```

Comment:
Clarify heals need to be partly "over-time" and not entirely up-front with the amount it gives, if not you end up with another issue.